docker-entrypoint.sh

```diff
@@ -20,4 +20,4 @@ do
 done
 
 echo  "Running: ansible-playbook $OPTS $@ playbooks/${PLAYBOOK}"
-exec ansible-playbook $OPTS $@ playbooks/${PLAYBOOK}
+ANSIBLE_STDOUT_CALLBACK=yaml exec ansible-playbook $OPTS $@ playbooks/${PLAYBOOK}
```

Comment:
I should have put a comment here about that. That is an Ansible mechanism that formats the task output so that it is readable, instead of a big unreadable blob. I discovered it while developing the work item and it helped a lot so I wanted to propose leaving it in. That is the only purpose for it, so removing it should not change any behavior it desired.

Reply:
I am good with that.